tests/nlp/utils/test_properties.py

```diff
@@ -83,30 +83,126 @@ def test_calculate_average_sentence_length_property(tweet_emotion_train_test_tex
     test_text = test.text
 
     # Act
-    result = calculate_default_properties(test_text, include_properties=['Average Sentence Length'])[0]
-    result_none_text = calculate_default_properties([None], include_properties=['Average Sentence Length'])[0]
+    result = calculate_builtin_properties(test_text, include_properties=['Average Sentence Length'])[0]
+    result_none_text = calculate_builtin_properties([None], include_properties=['Average Sentence Length'])[0]
+
+    # Assert
+    assert_that(result['Average Sentence Length'][0: 10], equal_to([6, 7, 11, 12, 8, 19, 3, 9, 12, 7]))
+    assert_that(result_none_text['Average Sentence Length'], equal_to([np.nan]))
 
     # Assert
     assert_that(result['Average Sentence Length'][0: 10], equal_to([6, 7, 11, 12, 8, 19, 3, 9, 12, 7]))
     assert_that(result_none_text['Average Sentence Length'], equal_to([np.nan]))
 
 
 def test_calculate_readability_score_property(tweet_emotion_train_test_textdata):
+
     # Arrange
     _, test = tweet_emotion_train_test_textdata
     test_text = test.text
 
     # Act
-    result = calculate_default_properties(test_text, include_properties=['Readability Score'])[0]
-    result_none_text = calculate_default_properties([None], include_properties=['Readability Score'])[0]
+    result = calculate_builtin_properties(test_text, include_properties=['Readability Score'])[0]
+    result_none_text = calculate_builtin_properties([None], include_properties=['Readability Score'])[0]
 
     # Assert
-    assert_that(result['Readability Score'][0: 10], equal_to([
-        102.045, 97.001, 80.306, 67.755, 77.103, 71.782, np.nan, 75.5, 70.102, 95.564
-    ]))
+    assert_that(result['Readability Score'][0: 10], equal_to([102.045, 97.001, 80.306, 67.755, 77.103, 
+                                                            71.782, np.nan, 75.5, 70.102, 95.564]))
     assert_that(result_none_text['Readability Score'], equal_to([np.nan]))
 
 
+def test_calculate_count_unique_urls(tweet_emotion_train_test_textdata):
```

Comment:
Please change the tests for the url & email tests to test on data that actually contains emails and urls. You can make up some simple data just to test it (I'm sure chatGPT will oblige)

Reply:
Done